it/src/test/java/org/corfudb/universe/GenericIntegrationTest.java

```diff
@@ -61,9 +64,9 @@ public Scenario getVmScenario(int numNodes) {
                 .universeParams(universeParams)
                 .build();
 
-        universe = UNIVERSE_FACTORY
-                .buildVmUniverse(universeParams, manager)
-                .deploy();
+        //Assign universe variable before deploy prevents resources leaks
```

Comment:
NIT: add a space after "//" and also copy this comment to getDockerScenario() before `UNIVERSE_FACTORY.buildDockerUniverse`

Reply:
done